README.md

```diff
@@ -52,6 +52,10 @@ applications enabled with oneDNN:
 * [API reference](https://oneapi-src.github.io/oneDNN/modules.html) provides
   a comprehensive reference of the library API.
 
+# Relevant articles
+- [Anatomy Of High-Performance Deep Learning Convolutions On SIMD Architectures](https://arxiv.org/abs/1808.05567) Evangelos Georganas, Sasikanth Avancha, Kunal Banerjee, Dhiraj Kalamkar, Greg Henry, Hans Pabst, Alexander Heinecke
+- [Applying the Roofline model for Deep Learning performance optimizations](https://arxiv.org/abs/2009.11224) Jacek Czaja, Michal Gallus, Joanna Wozna, Adam Grygielski, Luo Tao
+
```

Comment:
(random place)

I would suggest moving the "Relevant Publications" right before `# Support` section.
But this is a purely stylistic thing, so I would like to here the opinion of others.

On the other hand, I am not sure if we really want to have the publications on the main page. I checked github main page of some OSS projects (LLVM, TF, PyTorch, FBGEMM, FFTW, MxNet, TVM, ...) and almost nowhere found the publication section. So, maybe the alternative is to introduce a new section in the documentation (similar to what MLIR has: https://mlir.llvm.org/talks/). 

Tagging @vpirogov

Reply:
I moved "Relevant Publications" to pointed location . On  which document it should be part of, my thinking was like that:
If I were first time looking at this project I would check where it is used, github activity etc. and then seeing it has some related publications I would be additionally a bit more convinced that it is rock solid , industry and academia proven open source software.

Additionally I made this PR , because there were  two occasions when I was looking for article describing inner mechanism on oneDNN (Anatomy of High-Performance....) and remembered only that it was mentioned in issues. So each time it took a little bit too long to find it. So with this PR it will be very straight forward.

Looking forward to your opinions.

Reply:
I agree with @emfomenk suggestion to create a separate `Relevant Publications` page in documentation (I would suggest ideally before API References). I think this is the easiest way for the user to see the studies that use oneDNN. Plus, some additional content can be added to such page, including other papers, presentations, etc.